src/materialized/src/bin/materialized/main.rs

```diff
@@ -427,6 +459,26 @@ swap: {swap_total}KB total, {swap_used}KB used",
 
     sys::adjust_rlimits();
 
+    // Build Timely worker configuration.
+    let timely_worker = {
+        // This is a little goofy because `differential_dataflow::configure`
+        // wants the full `timely::Config` struct, when we only want to deal
+        // with `timely::WorkerConfig`. We just ignore the
+        // `timely::CommunicationConfig` that we create here, because we
+        // construct Timely's communication infrastructure separately.
```

Comment:
@frankmcsherry, not sure if there's a good way to fix this.

Reply:
Ah, we can probably make it more ergonomic in DD and then fold the change in. Is that best done before landing this to avoid introducing horrible stuff?

Reply:
I think either way is fine! Should we change DD to take a worker config as its first argument or did you Jane something else in mind?

Reply:
I'll change it that way, yeah.

Reply:
Ok, much nicer now, thanks!